config.yml

```diff
@@ -1,11 +1,11 @@
-default:
-  app_url: "http://localhost:8080/"
-  params_data_path: ".."
```

Comment:
Changing to . from .. will be storing the params folder inside the inputs_selection_app folder rather than the parent (inputs), have you made corresponding changes there?

Reply:
Reverted in subsequent commit.